src/main/java/io/github/thebusybiscuit/exoticgarden/items/FoodRegistry.java

```diff
@@ -1,531 +1,617 @@
+        14)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("NOODLES", "9174b34c549eed8bafe727618bab6821afcb1787b5decd1eecd6c213e7e7c6d", "&rNoodles", "", "&7&oRestores &b&o" + "5.0" + " &7&oHunger"),
+        new ItemStack[] {SlimefunItems.WHEAT_FLOUR, new ItemStack(Material.WATER_BUCKET), null, null, null, null, null, null, null},
+        10)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("BUTTERED_NOODLES", "9174b34c549eed8bafe727618bab6821afcb1787b5decd1eecd6c213e7e7c6d", "&rButtered Noodles", "", "&7&oRestores &b&o" + "6.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("NOODLES"), SlimefunItems.BUTTER, null, null, null, null, null, null, null},
+        12)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("MAC_N_CHEESE", "6bbecba5231805aaadda81d764b096eee62ed2e4cb447448544f5182b091f101", "&rMac n Cheese", "", "&7&oRestores &b&o" + "7.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("BUTTERED_NOODLES"), SlimefunItems.CHEESE, null, null, null, null, null, null, null},
+        14)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("MC_MUFFIN", "b8c34b9c7f568c229e5e2a7b914ddb6e710d1c095532d84019bf2de7d10cfd4e", "&rMc Muffin", "", "&7&oRestores &b&o" + "4.5" + " &7&oHunger"),
+        new ItemStack[] {new ItemStack(Material.BREAD), new ItemStack(Material.COOKED_PORKCHOP), SlimefunItems.CHEESE, new ItemStack(Material.EGG), null, null, null, null, null},
+        9)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("MC_NUGGETS", "11ec45cef70e846ff48ea174b60dc6ad2e2ab7a7ccf7eff32e914c53d5f3fc87", "&rMc Nuggets", "", "&7&oRestores &b&o" + "7.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), getItem("FRIED_CHICKEN"), null, null, null, null, null},
+        14)
+        .register(plugin);
+        
+        new CustomFood(food, new SlimefunItemStack("NUGGET_HAPPY_MEAL", "5f4984251a1b3d62dcbcd694b143e27d1850733f426087b860a45453a88b1481", "&rNugget Happy Meal", "", "&7&oRestores &b&o" + "8.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("MC_NUGGETS"), getItem("FRIES"), new ItemStack(Material.COOKIE), getItem("OAK_APPLE_JUICE"), null, null, null, null, null},
+        16)
+        .register(plugin);
+                
+        new CustomFood(food, new SlimefunItemStack("CHEESEBURGER_HAPPY_MEAL", "5f4984251a1b3d62dcbcd694b143e27d1850733f426087b860a45453a88b1481", "&rCheeseburger Happpy Meal", "", "&7&oRestores &b&o" + "8.5" + " &7&oHunger"),
+        new ItemStack[] {getItem("CHEESEBURGER"), getItem("FRIES"), new ItemStack(Material.COOKIE), getItem("OAK_APPLE_JUICE"), null, null, null, null, null},
+        17)
+        .register(plugin);

```

Comment:
Branded items, probably even trademarks.
So this right here could count as a violation of trademarks or an undisclosed product placement.
Not to mention the previously elaborated issues of scope and missing notices.